release-notes/Cloud/DCP_change_log_2025.md

```diff
@@ -9,6 +9,23 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 13 March 2025 - Enhancement - Admin - Tooltips for long node names [ID 42510]
+
+On the overview page of a DataMiner system, the labels of a system, and it's fail-over pair will now have a tooltip.
+Before longer node names would be clipped, causing users unable to see the last part of the node names.
```

Comment:
Is this what you mean here? It's not really clear to me why Failover pairs are explicitly mentioned right now.
```suggestion
In the Admin app, on the overview page for a DataMiner System, tooltips will now be shown for the titles of nodes in case these are too long to be displayed. Previously, long names were clipped, causing users to be unable to see the last part of the names.
```